roles/openshift-applier/tasks/process-template.yml

```diff
@@ -77,8 +77,8 @@
     {{ client }} {{ oc_action }} \
        {{ target_namespace }} \
        -f - \
-       {{ (oc_action == 'delete') | ternary(' --ignore-not-found', '') }} \
-       {{ (client == 'kubectl') | ternary(' --validate=false', '') }}
+       {{ (oc_action | regex_search('delete|get')) | ternary(' --ignore-not-found', '') }} \
```

Comment:
did you mean `patch` instead of `get` here? Also, don't think we should introduce `get` yet as that will be done as part of issue #134

Reply:
@pabrahamsson not sure if you saw this comment, but can we hold off on introducing `get` till we work #134? Reason for asking is that I don't think we'll support the `get` action at this point in the file, so it will be confusing for whoever implements it.

Reply:
Thanks @oybed, I've pushed a new update with `get` removed.

Reply:
great - thx. as soon as CI passes, we'll merge - I believe we should be in good shape now.